SwitchSpeak-iOS/SwitchSpeak-iOS/TouchGrid/Tree/TreeFactory.swift

```diff
@@ -15,16 +15,17 @@ class TreeFactory {
         grid size. this function is especially useful when you just
         want to build a tree from user settings.
      */
-    class func buildTree(type:ScanType, size:GridSize) -> Tree {
+    //dummy_num is the number of bummy button in this tree
+    class func buildTree(type:ScanType, size:GridSize, dummy_num: Int) -> Tree {
```

Comment:
Please change `dummy_num` from `snake_case` to `camelCase`. Python and C use `snake_case`, but most other modern languages (especially Swift) use `camelCase`